2021/23xxx/CVE-2021-23334.json

```diff
@@ -1,97 +1,99 @@
 {
```

Comment:
{} is not allowed for {'reference_data': [{'refsource': 'MISC', 'url': 'https://snyk.io/vuln/SNYK-JS-STATICEVAL-1056765', 'name': 'https://snyk.io/vuln/SNYK-JS-STATICEVAL-1056765'}, {'refsource': 'MISC', 'url': 'https://snyk.io/vuln/SNYK-JAVA-ORGWEBJARSNPM-1071860', 'name': 'https://snyk.io/vuln/SNYK-JAVA-ORGWEBJARSNPM-1071860'}, {'refsource': 'MISC', 'url': 'https://github.com/browserify/static-eval/blob/master/index.js%23L180', 'name': 'https://github.com/browserify/static-eval/blob/master/index.js%23L180'}]}